pynucastro/templates/amrexastro-cxx-microphysics/actual_rhs.H.template

```diff
@@ -236,6 +239,7 @@ void actual_jac(const burn_t& state, MatrixType& jac)
 
     // Account for the thermal neutrino losses
 
+#ifdef NEUTRINOS
     amrex::Real sneut, dsneutdt, dsneutdd, dsnuda, dsnudz;
```

Comment:
`dsneutdt` is used later, in line 270

Reply:
okay, I fixed that and the include and tested that it compiles now with `burn_cell` with the neutrinos disabled.